script_tests/tests/3_test-rpc-filter.js

```diff
@@ -0,0 +1,174 @@
+const expect = require("chai").expect;
+const assert = require("chai").assert;
+const Web3 = require("web3");
+const utils = require("./utils");
+const conf = require("./config.js");
+
+let currentFilterId = null;
+
+describe("Test filter API", function () {
+	afterEach(async () => {
+		if (currentFilterId) {
+			const res = await utils.customRequest("eth_uninstallFilter", [currentFilterId]);
+		}
+		currentFilterId = null;
+	});
+
+	it.skip("should return a number as hexstring eth_newBlockFilter", async function () {
+		const params = [];
+		currentFilterId = await utils.customRequest("eth_newBlockFilter", params);
+		assert.isNumber(currentFilterId);
+	});
+
+	it.skip("should return a number as hexstring eth_newPendingTransactionFilter", async function () {
+		const params = [];
+		currentFilterId = await utils.customRequest("eth_newPendingTransactionFilter", params);
+		assert.isNumber(currentFilterId);
+	});
+
+	it.skip("should return a number as hexstring when all options are passed with single address eth_newFilter", async function () {
+		const params = [
+			{
+				fromBlock: "0x1", // 1
+				toBlock: "0x2", // 2
+				address: "0xfd9801e0aa27e54970936aa910a7186fdf5549bc",
+				topics: ["0x01e0aa27e54970936aa910a713", "0x6aa910a7186fdf"],
+			},
+		];
+		currentFilterId = await utils.customRequest("eth_newFilter", params);
+		assert.isNumber(currentFilterId);
+	});
+
+	// it('should return a number as hexstring when all options are passed with address array', async function () {
+	//     const params = [{
```

Comment:
If all of the comment tests need to be skipped, please use `it.skip()` like what you did in other test files. Or delete.

Reply:
done